planning/behavior_path_start_planner_module/src/start_planner_module.cpp

```diff
@@ -1403,9 +1412,14 @@ void StartPlannerModule::setDebugData()
   // visualize collision_check_end_pose and footprint
   {
     const auto local_footprint = vehicle_info_.createFootprint();
+    std::map<PlannerType, double> collision_check_distances = {
+      {PlannerType::SHIFT, parameters_->shift_collision_check_distance_from_end},
+      {PlannerType::GEOMETRIC, parameters_->geometric_collision_check_distance_from_end}};
```

Comment:
(nits: this is duplicated process, maybe it is also good have as member)

Reply:
ah
define map as member variable you mean?
I agree

Reply:
yes because this map is like a parameter